src/utils/opt_in_checker.py

```diff
@@ -0,0 +1,260 @@
+# Copyright (C) 2018-2021 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+import os
+import time
+from enum import Enum
+from pathlib import Path
+from platform import system
+from sys import stdin
+
+from .colored_print import colored_print
+from .input_with_timeout import input_with_timeout
+
+
+class ISIPCheckResult(Enum):
+    DECLINED = 0
+    ACCEPTED = 1
+    NO_FILE = 2
+
+
+class DialogResult(Enum):
+    DECLINED = 0
+    ACCEPTED = 1
+    TIMEOUT_REACHED = 2
+
+
+class OptInChecker:
+    dialog_timeout = 50  # seconds
+    path_to_opt_in_out_script = os.path.dirname(os.path.dirname(os.path.realpath(__file__)))
+    opt_in_out_script_name = "opt_in_out"
+    doc_link = "docs.openvino.ai"
+    opt_in_out_script_run_command = "\'{} --opt_out\'".format(opt_in_out_script_name)
+    opt_in_question = "With your permission, Intel would like to collect occasional (anonymous) tool usage such " \
+                      "as software installation, development environment, and usage responses to improve our product " \
+                      "and your developer experience.  This confidential data will be kept a period of time " \
+                      "to study trends, and of course you can opt out at any time in the future by running {}. " \
+                      "We really appreciate your help by accepting (just type \"Y\" at the prompt) to show your " \
+                      "consent to the collection of this data, a \"N\" will prevent the anonymous data from " \
+                      "being sent back to Intel. " \
+                      "Thanks for replying with a \"Y\"!".format(opt_in_out_script_run_command)
+    opt_in_question = "Intel would like your permission to collect software performance and usage data for the " \
+                      "purpose of improving Intel products and services. This data will be collected directly " \
+                      "by Intel or through the use of Google Analytics. This data will be stored in countries " \
+                      "where Intel or Google operate.\nYou can opt-out at any time in the future by running {}.\n" \
+                      "More Information is available at {}.\n" \
+                      "Please type ‘Y’ to give your consent or ‘N’ to decline.".format(opt_in_out_script_run_command,
+                                                                                       doc_link)
```

Comment:
we have two different strings with the same variable name. please correct

Reply:
Done.